include/pokemon.h

```diff
@@ -579,7 +579,7 @@ const u32 *GetMonFrontSpritePal(struct Pokemon *mon);
 const u32 *GetMonSpritePalFromSpeciesAndPersonality(u16 species, u32 otId, u32 personality);
 const struct CompressedSpritePalette *GetMonSpritePalStruct(struct Pokemon *mon);
 const struct CompressedSpritePalette *GetMonSpritePalStructFromOtIdPersonality(u16 species, u32 otId , u32 personality);
-bool32 IsHMMove2(u16 move);
+bool8 IsMoveHm(u16 move);
```

Comment:
I'd probably call this `IsMoveHM` with a capital M, but admittedly we use both across the codebase (:sob:).

EDIT: In fact, `IsMoveHm` already existed and you just moved it. ffs! :joy:

Reply:
Yeah, haha. I removed the original `IsMoveHm` and turned `IsHMMove2` into the new `IsMoveHm` :eyes:

I'll rename `IsMoveHm` into `IsMoveHM`, I don't mind and it *is* consistent with its old label too *(`IsHMMove2`)*, so I'll push :eyes: